docs/scripts/update-all-docs.sh

```diff
@@ -111,3 +111,36 @@ echo ".. _list_toolchains:" > $VERSION_SPECIFIC_DIR/toolchains.rst
 echo "*(see also* \`\`eb --list-toolchains\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchains.rst
 echo >> $VERSION_SPECIFIC_DIR/toolchains.rst
 eb --list-toolchains --output-format rst >> $VERSION_SPECIFIC_DIR/toolchains.rst
+
+#  available toolchain options
+echo ".. _avail_toolchain_opts:" > $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo "*(see also* \`\`eb --avail-toolchain-opts <tcname>\`\` *)*" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+echo >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+IFS=$'\n'
+
+temp_tc="$(mktemp)"
+
+for i in `eb --list-toolchains | sed 1d | sort`
+do
+    tc=`echo $i | cut -d ':' -f1`
+    echo "${tc//[[:blank:]]/}" >> temp_tc
+done
+
+echo -n `cat temp_tc | sed -n 1p`"_" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+
+cat temp_tc | sed 1d | while read line
+do
+    echo -n " - ""$line""_" >> $VERSION_SPECIFIC_DIR/toolchain_opts.rst
+done
+
```

Comment:
@Caylo you need to emit an empty line here with `echo` (without arguments, otherwise you break the first anchor, cfr. http://boegel-eb.readthedocs.io/en/test/version-specific/toolchain_opts.html